src/components/dropdown/dropdown.scss

```diff
@@ -15,6 +15,7 @@
     z-index: 4;
     font-weight: 600;
     font-size: rem(14px);
+    outline: none;
```

Comment:
@iamstarkov What if we remove it from the dropdown input (as this is not implemented consistently on other inputs), and keep it for the range selector, but with a blue background? Like the attached image? 
![image](https://cloud.githubusercontent.com/assets/2117394/21306194/ae6044b4-c5c7-11e6-9052-e3c6b4db4393.png)